eduvpn/ovpn.py

```diff
@@ -1,24 +1,132 @@
-import re
+from io import StringIO
+from typing import Optional, Callable, Iterable, List
+from gettext import gettext as _
 
 
-# matches for example: remote vpn.example.com 1234 udp
-# NOTE: multiline-mode does not consider \r
-#       so we must match for that explicitly.
-FIND_REMOTE_UDP_LINES = re.compile(
-    r'^(remote +\S+ +[0-9]+ +udp\r?)$',
-    re.MULTILINE
-)
+class Item:
+    def to_string(self) -> str:
+        raise NotImplementedError
```

Comment:
so this is not supposed to be called?

Reply:
Not on this type, but it's part of the interface. The typechecker doesn't seem to care, but I like to make it explicit as described in the [docs](https://docs.python.org/3/library/exceptions.html#NotImplementedError).